src/main/scala/wdl4s/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -114,6 +116,23 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WdlValue] {
     extractArguments flatMap arrayLength
   }
 
+  def prefix(params: Seq[Try[WdlValue]]): Try[WdlArray] = {
+    def extractTwoArguments: Try[(WdlValue, WdlValue)] = params.size match {
+      case 2 => (params.head |@| params.tail.head) map { (_, _) }
+      case n => Failure(new UnsupportedOperationException(s"prefix() expects two parameters but got $n"))
+    }
+
+    val makePrefixedString = (prefixString: WdlValue, strings: WdlValue) => (prefixString, strings) match {
+      case (WdlString(p), WdlArray(WdlArrayType(WdlStringType), ss)) =>
+        val result = ss map { s => WdlString(p + s.valueString) }
+        Success(WdlArray(WdlArrayType(WdlStringType), result))
+      case (x, y) =>
+        Failure(new UnsupportedOperationException(s"prefix() expects a String and Array[String] parameters but got (${prefixString.wdlType.toWdlString}, ${strings.wdlType.toWdlString})"))
```

Comment:
Inconsistent format, would prefer `The function prefix expects arguments (X, Y) but got (X', Y')`